python/tvm/runtime/module.py

```diff
@@ -185,6 +185,11 @@ def type_key(self):
         """Get type key of the module."""
         return _ffi_api.ModuleGetTypeKey(self)
 
+    @property
+    def format(self):
+        """Get the format of the module."""
+        return _ffi_api.ModuleGetFormat(self)
+
     def get_source(self, fmt=""):
```

Comment:
i think Module can sometimes have multiple formats (e.g. CUDA modules do this I believe). what about leveraging this and modifying export_module to try with fmt="c" here and if not also try fmt="cc"?